bubuku/zookeeper.py

```diff
@@ -118,5 +150,143 @@ def take_lock(self, *args, **kwargs):
                 _LOG.error('Failed to obtain lock for exhibitor, retrying', exc_info=e)
 
 
-def load_exhibitor(initial_hosts: list, zookeeper_prefix):
-    return Exhibitor(initial_hosts, 8181, zookeeper_prefix)
+class BukuExhibitor(object):
```

Comment:
The name is questionable as for me. I thought we wanted to abstract from Exhibitor. I think the better name would be something like ZkService.
And I think we can put it into separate file. This one grows too big.